### PR TITLE
Exclude RouteTables created by TCNP from private table lookup

### PR DESCRIPTION
**Pull request by kubasobon**
## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: service/controller/resource/cproutetables/resource.go [hunk 1]
```diff
@@ -44,6 +44,11 @@ func New(config Config) (*Resource, error) {
 		return nil, microerror.Maskf(invalidConfigError, "%T.Installation must not be empty", config)
 	}
 
+	// TODO: This is temporary fix until we get rid of routeTables flag.
```
Comment (wtty-fool) [inline service/controller/resource/cproutetables/resource.go:47]: Note: remember to write something more informative here or fix it in config parsing where it's split by `strings.Split`

**Comment by xh3b4sd** [comment]
Hey Kuba, is this PR still relevant?

**Comment by calvix** [comment]
@kubasobon what about this PR seem really old, do we need it?

**Comment by calvix** [comment]
no reaction, cleaning up

**Pull request closed without merging**